CMSIS/NN/Include/arm_nnfunctions.h

```diff
@@ -1645,6 +1645,22 @@ void arm_softmax_with_batch_q7(const q7_t * vec_in, const uint16_t nb_batches,co
                                                 const int32_t out_shift,
                                                 const int32_t out_mult);
 
+/**
+ * @defgroup Reshape Reshape Functions
+ *
+ */
+
+   /**
+    * @brief Reshape a s8 vector into another with different shape
+    * @param[in]  input      points to the s8 input vector
+    * @param[out] output     points to the s8 output vector
```

Comment:
Please add an additional comment that we expect the output to be a non-overlapping memory area compared to the input.